docs/source/dev_guide/algorithms/implicit_als/computation-distributed-training.rst

```diff
@@ -0,0 +1,246 @@
+.. ******************************************************************************
+.. * Copyright 2020 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+.. _implicit_als_distributed_training:
+
+Distributed Processing: Training
+================================
+
+The distributed processing mode assumes that the data set is split in ``nblocks`` blocks across computation nodes.
+
+.. contents::
+    :local:
+    :depth: 1
+
+Algorithm Parameters
+********************
+
+At the training stage, implicit ALS recommender in the distributed processing mode has the following parameters:
+
```

Comment:
It seems the table with parameters is missing here.

Reply:
fixed, thanks